### ISSUE TITLE
Missing color_normal_mean.hdf5 for pretrained model

### ISSUE THREAD

**Issue by ChemJeff** [DLR-RM/SingleViewReconstruction#1]
Hi Maximilian,
I am trying to predict some scene reconstruction using your pretrained model. However, a "color_normal_mean.hdf5" file is needed to do correct normalization to my own feeded images as done in the training. Since I have no idea the mean of colors and normals in the training data used for the released model, I would appreciate if you could upload the color_normal_mean.hdf5 which corresponds to your pretrained model.
Thanks!

**Comment by themasterlink** [comment]
Hey,

sorry about that, I will upload them and provide a link here.

Best,
Max

**Comment by themasterlink** [comment]
Here is a link to the two hdf5 container: https://dlrmax.dlr.de/get/ece5e2ed-f561-5bac-ad09-ec25da95b486/ 

I will add this to the download script.

**Comment by themasterlink** [comment]
Does this solve this issue?

**Comment by themasterlink** [comment]
The missing file is now in the `data` folder.
